test/LowerColorado_TX_v4/test_AnA_V4_HYFeature.yaml

```diff
@@ -67,7 +67,8 @@ compute_parameters:
         streamflow_da:
             #----------
             streamflow_nudging            : True
-            diffusive_streamflow_nudging  : False                    
+            diffusive_streamflow_nudging  : False  
+            lastobs_file                  : lastobs/nudgingLastObs.2023-04-02_01:00:00.nc                 

```

Comment:
Are these permanent changes for this config file? If not, can you undo these?

Reply:
I commented out lastobs_file.